docs/index.asciidoc

```diff
@@ -21,7 +21,9 @@ include::{include_path}/plugin_header.asciidoc[]
 
 ==== Description
 
-This output lets you send events to the Elastic App Search solution.
+This output lets you send events to the Elastic App Search solution, 
+both the https://www.elastic.co/downloads/app-search[Self-Managed] or 
```

Comment:
nitpick: the mix of capital-words and hyphenation in `Self-Managed` vs `managed service` feels inconsistent.